### PR TITLE
Refactors files version app commands.

### PR DESCRIPTION
**Pull request by fsamapoor**
## Summary
I have made some adjustments to the apps/files_version/lib/Command classes to improve the code readability.

The improvements in this PR include but are not limited to:
- Using PHP8's constructor property promotion
- Adding return types
- Replacing return code integers with more readable strings.
- Using early returns

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: apps/files_versions/lib/Command/CleanUp.php [hunk 1]
```diff
@@ -85,46 +75,45 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
 	}
 
 
 	/**
 	 * delete versions for the given user
-	 *
-	 * @param string      $user
-	 * @param string|null $path
 	 */
 	protected function deleteVersions(string $user, string $path = null): void {
```
Comment (come-nc) [inline apps/files_versions/lib/Command/CleanUp.php:118]:
```suggestion
	protected function deleteVersions(string $user, ?string $path = null): void {
```
Better to be explicit

File: apps/files_versions/lib/Command/Expire.php [hunk 2]
```diff
@@ -62,7 +53,7 @@ public function handle() {
 			// In case of external storage and session credentials, the expiration
 			// fails because the command does not have those credentials
 
-			/** @var LoggerInterface */
+			/** @var LoggerInterface $logger */
```
Comment (come-nc) [inline apps/files_versions/lib/Command/Expire.php:56]:
```suggestion
```

**Review by come-nc — COMMENTED** [review]
:+1:

File: apps/files_versions/lib/Command/CleanUp.php (hunk 1 above)
Comment (fsamapoor) [reply apps/files_versions/lib/Command/CleanUp.php]: Sure.

**Review by come-nc — APPROVED** [review]

**Pull request merged**